cmake/modules/falcosecurity-libs-repo/patch/grpc.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/cmake/modules/grpc.cmake b/cmake/modules/grpc.cmake
+index ffa4e6c4..346939b9 100644
+--- a/cmake/modules/grpc.cmake
++++ b/cmake/modules/grpc.cmake
+@@ -125,6 +125,7 @@ else()
+ 				-DZLIB_ROOT:STRING=${ZLIB_SRC}
+ 			BUILD_IN_SOURCE 1
+ 			BUILD_BYPRODUCTS ${GRPC_LIB} ${GRPCPP_LIB} ${GPR_LIB} ${GRPC_LIBRARIES}
++			INSTALL_COMMAND DESTDIR=/ make install
```

Comment:
N.B. 
the resulting installation directory is computed by `DESTDIR` + `CMAKE_INSTALL_PREFIX`.
Basically, it forces gRPC installation files to be always installed into the path specified by `CMAKE_INSTALL_PREFIX` that's set to `${GRPC_SRC}/target` (see [here](https://github.com/falcosecurity/libs/blob/master/cmake/modules/grpc.cmake#L94)).

See also:
- https://cmake.org/cmake/help/latest/envvar/DESTDIR.html
https://cmake.org/cmake/help/latest/variable/CMAKE_INSTALL_PREFIX.html